### PR TITLE
appended '- 400' to the 400 error page title for distinction from 500 in analytics

### PR DESCRIPTION
**Pull request by ChristopherMShelton**
# Motivation and Context
So that we can track 400 and 500 errors independently in google analytics

# What has changed
added - 400 to the 400 template title, this is what our user researcher deemed most appropriate.

# How to test?
hit the error and look at the title

# Links
https://trello.com/c/JllhbBP2/1069-400-and-500-page-both-using-the-same-title-1

### CODE REVIEW

**Comment by insacuri** [comment]
Are we not doing the same for the 500 page?

**Comment by ChristopherMShelton** [comment]
> Are we not doing the same for the 500 page?

yea probably makes sense to be honest, otherwise analytics will be convoluted historically from this - this way both start getting tracked fresh

**Comment by ChristopherMShelton** [comment]
> Cannot approve as tests are not passing locally

this isn't a change request

File: frontstage/templates/errors/400-error.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% import 'partials/section.html' as section %}
 {% extends "layouts/_twocol.html" %}
 
-{% block page_title %}An error has occurred - ONS Business Surveys{% endblock %}
+{% block page_title %}An error has occurred - ONS Business Surveys - 400){% endblock %}
```
Comment (necrophonic) [inline frontstage/templates/errors/400-error.html]: I'd be potentially inclined to say `4xx` (and `5xx` below) as these'll pop up for a variety of those error types.

Is there anyway the actual code can be pushed into the template? Might be useful to have breakdowns of the _actual_ error codes in analytics?

Comment (ChristopherMShelton) [reply frontstage/templates/errors/400-error.html]: Told not to have any technical jargon in the templates, this is just so we can see which pages are being hit and how often in the analytics. Also fairly sure once it arrives at frontstage they are in fact all 500's or 400's (except specifically 404 which has a different template)

Comment (necrophonic) [reply frontstage/templates/errors/400-error.html]: Fair enough 👍

**Pull request merged**